csbuild/_build/__init__.py

```diff
@@ -126,13 +126,28 @@ def _dependenciesMet(buildProject, tool):
 def _getGroupInputFiles(buildProject, tool):
 	with perf_timer.PerfTimer("Collecting group inputs"):
 		fileList = ordered_set.OrderedSet()
-		for inputFile in tool.inputGroups:
-			log.Info("Checking if all builds for {} are done yet", inputFile)
-			if buildProject.toolchain.IsOutputActive(inputFile):
-				log.Info("Extension {} is still active, can't build yet.", inputFile)
-				return None
-			log.Info("{} is ok to build.", inputFile)
-			fileList.update([x for x in buildProject.inputFiles.get(inputFile, []) if not x.WasToolUsed(tool)])
+		if tool.crossProjectInputGroups:
+			for inputFile in tool.crossProjectInputGroups:
+				log.Info("Checking if all cross-porject builds for {} are done yet", inputFile)
```

Comment:
Spelling error: "cross-porject"

If you copied this string from somewhere else, you may want to do a search&replace.